ow_power_system/src/power_system_node.cpp

```diff
@@ -5,18 +5,68 @@
 // OW power system ROS node - publishes values from a csv generated by the matlab
 // battery model as a placeholder for once battery models are linked.
 
+
+#include <chrono>
+
 #include <ros/ros.h>
 #include <ros/package.h>
 #include <ros/console.h>
 #include <std_msgs/Float64.h>
-//CSV handling
-#include <vector>
-#include <iostream>
-#include <fstream>
-#include <sstream>
+
+#include "power_system_util.h"
 
 using namespace std;
-vector<vector<double>> loadCSV(const std::string& filename);
+
+
+// This example sets up a simple synchronos prognoser to predict battery end of
+// discharge using a Monte Carlo predictor and an Unscented Kalman Filter.
+int simple_example() {
```

Comment:
I don't see this function used anywhere.  Was it supposed to be?

Reply:
It was. There was a miscommunication on my part with some of the changes Ussama helped me make. I thought this function was integrated and produced the resulting output for the power system node, because I had formerly gotten null output when running in the past. It turns out that this function had not actually been invoked, which explains the matching behavior you've seen post integration.

Reply:
I think it is likely best to get this PR merged and create a separate PR for calling this function to update the output.